### PR TITLE
[RAPTOR-3576] bump tensoflow versions to 2.2.1 and unify them across the repo

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Unify and bring tensorflow versions across the repo to 2.2.1, to resolve the issue.
https://github.com/tensorflow/tensorflow/security/advisories/GHSA-p2cq-cprg-frvm

Still have to keep keras in the env and in keras_predictor, until @rsugumar cheks what is going on with pickling `tf.keras` model as joblib.

### CODE REVIEW

**Comment by yakov-g** [comment]
Hey @rsugumar ,
I have issues with moving `./model_templates/training/python3_keras_vizai_joblib` to use tensorflow keras. When i change all the keras modules to be imported from tensorflow, I get the following error: `TypeError: can't pickle _thread.RLock objects`
Otherwise all the exemples: inference and training work.

File: model_templates/training/python3_keras_vizai_joblib/model_utils.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from typing import Tuple
 
 # keras imports
+# I was not able to switch this to tensoflow keras as it failed on pickling joblib
```
Comment (yakov-g) [inline model_templates/training/python3_keras_vizai_joblib/model_utils.py:4]: @rsugumar here

**Review by klichukb — APPROVED** [review]

**Review by meltzg — APPROVED** [review]

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**6.12** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**